_topic_map.yml

```diff
@@ -283,7 +283,7 @@ Topics:
 - Name: Disconnected installation
   File: disconnected_install
   Distros: openshift-enterprise
-- Name: InInstalling a stand-alone deployment of OpenShift container registry
+- Name: InInstalling a stand-alone deployment of OpenShift container image registry
```

Comment:
Looks like a long standing typo :)

Reply:
Luckily it's just in 3.11, so this PR will cover it. :)